src/applications/mhv-medications/reducers/prescriptions.js

```diff
@@ -106,6 +106,12 @@ export const prescriptionsReducer = (state = initialState, action) => {
         apiError: false,
       };
     }
+    case Actions.Prescriptions.CLEAR_FILL_NOTIFICATION: {
```

Comment:
would you be able to add a test case for this as well?

Reply:
TY. done!